packages/cli/src/commands/env/add.ts

```diff
@@ -0,0 +1,48 @@
+import * as api from '../../rest/api'
+import { Flags, Args, ux } from '@oclif/core'
+import { AuthCommand } from '../authCommand'
+
+export default class EnvAdd extends AuthCommand {
```

Comment:
It would also be nice to have a way to update environment variables.

Reply:
@clample, this totally makes sense. I just added this command.